apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift

```diff
@@ -52,7 +52,7 @@ struct GroupMemberInfoView: View {
                     .listRowBackground(Color.clear)
 
                 Section {
-                    if let contactId = member.memberContactId {
+                    if let contactId = member.memberContactId, member.memberActive {
                         if let chat = chatModel.getContactChat(contactId),
                            chat.chatInfo.contact?.directContact ?? false {
```

Comment:
This condition also needs to change to allow sending to indirect contacts if there are messages with them.

Reply:
they're used in this case and this condition is true